test/test_core.py

```diff
@@ -1499,8 +1495,6 @@ def test_exceptions_longjmp2(self):
 
   @with_all_eh_sjlj
   def test_exceptions_longjmp3(self):
-    if '-fwasm-exceptions' in self.emcc_args:
-      self.skipTest('https://github.com/emscripten-core/emscripten/issues/17004')
```

Comment:
To be precise this was not fixed by this PR. It was already working under lsan and asan. Not sure what the cause for the failure was. Also many tests reported in #14035 were already working too.

And we were disabling this test as a whole and not only under lsan! 😨